docs/src/mdx-components.tsx

```diff
@@ -9,25 +9,100 @@ import {
   TableRow,
   TableCell,
 } from "@kiwicom/orbit-components";
-import { InlineCode, Pre } from "./components/Code";
+import { Link } from "gatsby";
+import { css } from "styled-components";
+import { InlineCode, CodeBlock } from "./components/Code";
 
-interface Props {
-  children: React.ReactNode;
-}
-
-export const p = Text;
-export const h1 = (props: Props) => <Heading as="h1" type="display" {...props} />;
-export const h2 = (props: Props) => <Heading as="h2" type="title1" {...props} />;
-export const h3 = (props: Props) => <Heading as="h3" type="title2" {...props} />;
-export const h4 = (props: Props) => <Heading as="h4" type="title3" {...props} />;
-export const h5 = (props: Props) => <Heading as="h5" type="title4" {...props} />;
-export const h6 = (props: Props) => <Heading as="h6" type="title5" {...props} />;
-export const table = Table;
-export const thead = TableHead;
-export const tbody = TableBody;
-export const tr = TableRow;
-export const td = TableCell;
-export const th = TableCell;
-export const pre = Pre;
+export const p = ({ children }: React.HTMLAttributes<HTMLParagraphElement>) => (
+  <Text>{children}</Text>
+);
+export const h2 = ({ children }: React.HTMLAttributes<HTMLHeadingElement>) => (
+  <Heading as="h2" type="title1">
+    {children}
+  </Heading>
+);
+export const h3 = ({ children }: React.HTMLAttributes<HTMLHeadingElement>) => (
+  <Heading as="h3" type="title2">
+    {children}
+  </Heading>
+);
+export const h4 = ({ children }: React.HTMLAttributes<HTMLHeadingElement>) => (
+  <Heading as="h4" type="title3">
+    {children}
+  </Heading>
+);
+export const h5 = ({ children }: React.HTMLAttributes<HTMLHeadingElement>) => (
+  <Heading as="h5" type="title4">
+    {children}
+  </Heading>
+);
+export const h6 = ({ children }: React.HTMLAttributes<HTMLHeadingElement>) => (
+  <Heading as="h6" type="title5">
+    {children}
+  </Heading>
+);
+export const table = ({ children }: React.TableHTMLAttributes<HTMLTableElement>) => (
+  <Table>{children}</Table>
+);
+export const thead = ({ children }: React.HTMLAttributes<HTMLTableSectionElement>) => (
+  <TableHead>{children}</TableHead>
+);
+export const tbody = ({ children }: React.HTMLAttributes<HTMLTableSectionElement>) => (
+  <TableBody>{children}</TableBody>
+);
+export const tr = ({ children }: React.HTMLAttributes<HTMLTableRowElement>) => (
+  <TableRow>{children}</TableRow>
+);
+export const td = ({
+  children,
+  align,
+  valign,
+}: React.TdHTMLAttributes<HTMLTableDataCellElement>) => (
+  <TableCell
+    as="td"
+    align={align === "left" || align === "center" || align === "right" ? align : undefined}
+    verticalAlign={valign}
+  >
+    {children}
+  </TableCell>
+);
+export const th = ({ children, align }: React.ThHTMLAttributes<HTMLTableHeaderCellElement>) => (
+  <TableCell
+    as="th"
+    align={align === "left" || align === "center" || align === "right" ? align : undefined}
```

Comment:
why undefined? if default value for `TableCell` is `left` 🤔

Reply:
Passing `undefined` allows the default value to kick in, I didn't want to copy defaults here.